src/main/java/net/discordjug/javabot/util/MessageActionUtils.java

```diff
@@ -1,21 +1,20 @@
 package net.discordjug.javabot.util;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.CompletableFuture;
-import java.util.function.Consumer;
-
+import club.minnced.discord.webhook.receive.ReadonlyMessage;
```

Comment:
Please avoid unnecessary changes to classes. Your PR doesn't modify that class except by reordering imports.